discovery-infra/tests/base_test.py

```diff
@@ -18,18 +20,27 @@
 from tests.conftest import env_variables, qe_env
 from download_logs import download_logs
 
-
 class BaseTest:
+    @staticmethod
+    def override_vm_parameters(**kwargs):
```

Comment:
where it is used?

Reply:
in the tests

Reply:
moved the calling of this into `nodes` fixture